Locker/Helpers/Devices.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Devices.swift
+//  Locker
+//
+//  Created by Zvonimir Medak on 20.10.2021..
+//
+
+import Foundation
+
+class Devices {
+    static let shared = Devices()
+
+    private(set) var devicesURL: URL? = Constants.devicesURL
+
+    private init() {}
+
+    func fetchDevices() {
+        guard let devicesURL = devicesURL else {
```

Comment:
For now, we will not expose this URL (read/write). Can you remove property and just use that constant here?

Reply:
I still left the guard let with the URL from Constants since it is optional if that's what you meant

Reply:
Yes, `guard let` part is ok :)